cql3/functions/functions.cc

```diff
@@ -684,6 +684,20 @@ function_call::raw::to_string() const {
     return format("{}({})", _name, join(", ", _terms));
 }
 
+namespace aggregate_fcts {
+
+template <typename Type>
+void impl_sum_function_for<Type>::validate_sum() {
```

Comment:
We should move all of this code to the .cc file, but having a template declaration in a header and the definition in a .cc file is asking for trouble. Please keep this in the .hh for now.

I will send a RFC patch doing the move and rebase on top of this once it is in.

Reply:
makes sense